libraries/botbuilder-core/botbuilder/core/channel_service_handler.py

```diff
@@ -343,6 +351,24 @@ def __init__(
         """
         raise BotActionNotImplementedError()
 
+    async def on_get_conversation_member(
+        self, claims_identity: ClaimsIdentity, conversation_id: str, member_id: str,
+    ) -> ChannelAccount:
+        """
+        get_conversation_member() API for Skill.
```

Comment:
'for Skill'?

Reply:
Copy pasta from get_conversation_members(), actually have no idea why it is there

Reply:
looked at this some more... since it is in channel_service_handler, it IS part of the api for a skill.